src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_params.py

```diff
@@ -180,3 +180,19 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
     with self.argument_context('iot hub manual-failover') as c:
         c.argument('failover_region', options_list=['--failover-region', '--fr'], help='The region that the IoT hub'
                    'fails over to. Must be the paired region to the current IoT hub region.')
+
+    # Arguments for IoT Digital Twin
+    with self.argument_context('iot pnp') as c:
+        c.argument('repo_endpoint', options_list=['--repo-rp', '--rp', '--endpoint', '-e'], help='PnP endpoint.')
+        c.argument('repo_id', options_list=['--repo-id', '--id'], help='Repository Id.')
+
+    with self.argument_context('iot pnp repository') as c:
+        c.argument('repo_name', options_list=['--repo-name', '--name', '-n'], help='Repository Name.')
+
+    with self.argument_context('iot pnp repository get-provision-status') as c:
+        c.argument('track_id', options_list=['--track-id', '--tid'], help='Tracking id (provisioningState).')
+
+    with self.argument_context('iot pnp authkey') as c:
+        c.argument('key_id', options_list=['--key-id', '--kid'], help='Key Id.')
+        c.argument('user_role', options_list=['--user-role', '--role', '--ur', '-r'], help='User role for the key.',
```

Comment:
`--role` and `-r` are fine.

Reply:
keeping only --role as -r we are using for repository also